core/camel-main/src/main/java/org/apache/camel/main/BaseMainSupport.java

```diff
@@ -145,7 +145,8 @@ public abstract class BaseMainSupport extends BaseService {
     protected final OrderedLocationProperties wildcardProperties = new OrderedLocationProperties();
     protected RoutesCollector routesCollector = new DefaultRoutesCollector();
     protected String propertyPlaceholderLocations;
-    protected String defaultPropertyPlaceholderLocation = MainConstants.DEFAULT_PROPERTY_PLACEHOLDER_LOCATION;
+    protected String defaultPropertyPlaceholderLocation
+            = MainConstants.DEFAULT_PROPERTY_PLACEHOLDER_LOCATION + "," + MainConstants.DEFAULT_OBSERVABILITY_SERVICES_PROPERTY_LOCATION ;
```

Comment:
This is required in order to load the configuration properties when available on classpath.